test/utils/utils.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:lotti/utils/platform.dart';
+import 'package:media_kit/src/libmpv/core/native_library.dart';
+
+void ensureMpvInitialized() {
+  if (isMacOS) {
+    NativeLibrary.ensureInitialized(libmpv: '/opt/homebrew/bin/mpv');
```

Comment:
media_kit bundles libmpv within `media_kit_libs_macos_audio`, by the way.
I'm not sure how if using external version will be a good idea.

Reply:
Ah nice, that means I can remove that hack after updating the lib? Awesome, thanks for fixing it, felt like an odd hack having to do that to make the tests pass. Should have filed an issue right away :]